python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -879,6 +896,22 @@ def _create_pattern_spokes(self, src_ctr, src_delta_array, src_dist_array,
                 n_fail += 1
                 continue
 
+            # Plane project the candidate source spoke and compute the cosine
+            # and sine of the opening angle.
+            proj_src_delta = (
+                src_delta_array[src_idx] -
+                np.dot(src_delta_array[src_idx], src_ctr) * src_ctr)
+            src_geom_dist = np.sqrt(
+                np.dot(proj_src_delta, proj_src_delta) *
+                proj_src_ctr_dist_sq)
```

Comment:
src_geom_dist should be dist_geom_src by DM C++ style guide (3.1.0)

Reply:
Changed the name to geom_dist_src/ref. geom_dist for "geometric distance".